src/scene/vcMedia.cpp

```diff
@@ -310,6 +310,14 @@ void vcMedia::HandleSceneExplorerUI(vcState *pProgramState, size_t *pItemID)
   }
 }
 
+void vcMedia::HandleSceneEmbeddedUI(vcState * /*pProgramState*/)
+{
+  const char *imageTypeNames[] = {vcString::Get("scenePOILabelImageTypeStandard"), vcString::Get("scenePOILabelImageTypeOriented"), vcString::Get("scenePOILabelImageTypePanorama"), vcString::Get("scenePOILabelImageTypePhotosphere"), vcString::Get("scenePOILabelImageTypeScreen")};
```

Comment:
We normally break these over multiple lines - otherwise, there should be a space after `{` and before `}`.